runtime/doc/filetype.txt

```diff
@@ -512,30 +512,42 @@ Local mappings:
 
 MAN							*ft-man-plugin* *:Man*
 
-Displays a manual page in a nice way.  Also see the user manual
-|find-manpage|.
-
-To start using the ":Man" command before any manual page was loaded, source
-this script from your startup vimrc file: >
-
-	runtime ftplugin/man.vim
-
-Options:
-'iskeyword'	the '.' character is added to be able to use CTRL-] on the
-		manual page name.
+View manpages from the comfort of vim. Features include syntax highlighting,
+autocompletion, multilingual support, and manpage jumping.
+Also see |find-manpage|.
 
 Commands:
-Man {name}	Display the manual page for {name} in a window.
-Man {number} {name}
-		Display the manual page for {name} in a section {number}.
-
-Global mapping:
-<Leader>K	Displays the manual page for the word under the cursor.
+Man			Display the manual page for <cWORD> in a new tab.
+Man {name}		Display the manual page for {name} in a new tab.
+Man {sect} {name}	Same as above except specify the section.
+Man {name}({sect})	Alternate syntax to above.
```

Comment:
`man` doesn't support this syntax, so why should `:Man`? This seems like feature bloat.